static/app/utils/queryClient.tsx

```diff
@@ -115,7 +115,7 @@ function useApiQuery<TResponseData, TError = RequestError>(
     data: data?.[0],
     getResponseHeader: data?.[2]?.getResponseHeader,
     ...rest,
-  };
+  } as UseApiQueryResult<TResponseData, TError>;
```

Comment:
I wanted to just override `data`, but the react query typings are very complex and doing so would have removed the ability to assert that `data` exists when doing `if (isLoading) { return null }`